packages/core-web/src/index.js

```diff
@@ -89,6 +89,47 @@ window.handleSiteNavClick = function (elem, useAnchor = true) {
   dropdownIcon.classList.toggle('site-nav-rotate-icon');
 };
 
+window.handleScrollTop = function () {
+  document.body.scrollIntoView({ block: 'start', behavior: 'smooth' });
+};
+
+function setScrollTopButtonOpacity(scrollTopButton, opacity = '40%') {
+  scrollTopButton.style.setProperty('--scroll-top-button-opacity', opacity);
+}
+
+function displayScrollTopButton(scrollTopButton) {
+  if (document.body.scrollTop > 20 || document.documentElement.scrollTop > 20) {
+    scrollTopButton.style.display = 'block';
```

Comment:
would it make sense to move `setScrollTopButtonOpacity` in here instead? since setting opacity before `display: block` has no effect

then in `timer.lightenButtonTimer` you'd just set the opacity as well